pkg/openstack/openstack.go

```diff
@@ -483,7 +485,7 @@ func (os *OpenStack) Routes() (cloudprovider.Routes, bool) {
 		return nil, false
 	}
 
-	r, err := NewRoutes(os, network, netExts["extraroute-atomic"], netExts["allowed-address-pairs"])
+	r, err := NewRoutes(os, network, netExts["extraroute-atomic"], netExts["allowed-address-pairs"], os.routeOpts.AutoConfigSecurityGroup)
```

Comment:
the `os` is already passed to the `NewRoutes` func. There is no need to pass the `os.routeOpts.AutoConfigSecurityGroup`.